starknet-foundry/crates/cast/src/helpers/scarb_utils.rs

```diff
@@ -31,30 +31,34 @@ fn get_property(
             )),
         None => profiled
             .and_then(|t| t.get(property))
-            .and_then(serde_json::Value::as_str)
+            .and_then(Value::as_str)
             .map(String::from)
             .ok_or(anyhow!("Property not found in tool: {property}")),
     }
 }
 
 pub fn parse_scarb_config(
     profile: &Option<String>,
-    path: Option<Utf8PathBuf>,
+    path: &Option<Utf8PathBuf>,
```

Comment:
I would rename it to `cli_manifest_path` or `cli_path`, the `path` name doesn't really tell much

Reply:
Hmm, I would say that path inside the `parse_scarb_config` function tells enough 😅  but can change it if you want